### ISSUE TITLE
audit-avt1-homepage: form combobox needs label

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#1410]
## Environment

> Operating system: Mac
> Browser: Chrome
> AVT :one:
> Page: [Homepage](https://www.carbondesignsystem.com/)

## Detailed description: 

Form control with "combobox" role has no associated label
Each form control must have an associate

## What to do: 

- Add a for attribute to a visible label with the id of this control (e.g. <label for="controlID">);
- OR put the UI element inside a visible <label> element;
- OR add an aria-labelledby attribute to the control. It must point to visible text on the page that is meaningful as a label;
- OR only if the design cannot have a visible label, use the title attribute to provide a label.

## Why is this important?
Associating a meaningful label with every UI control allows the browser and assistive technology to expose and announce the control to a user. Associating a visible label also provides a larger clickable area.
